### ISSUE TITLE
Bug report: gtsummary v 2.0.0 (and 2.0.1) no longer supports add_glance_table() for mice objects

### ISSUE THREAD

**Issue by CharlyMarie** [ddsjoberg/gtsummary#1912]
Brief description of the problem: gtsummary v 2.0.0 (and 2.0.1) no longer supports _add_glance_table()_ and _add_glance_source_note()_ for mice objects.

It was included in earlier versions of gtsummary (#1137), but seems to be broken.

``` r
library("tidyverse")
library("mice")
library("gtsummary")

imp <- mice(nhanes2, print = FALSE, maxit = 1)

with(imp, lm(bmi ~ age)) %>% 
  pool() %>% 
  broom::glance()
#>   nimp nobs r.squared adj.r.squared
#> 1    5   25  0.269706      0.202442

with(imp, lm(bmi ~ age)) %>%
  tbl_regression()
```

<div id="nwfgkjwaxt" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">

<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1" scope="col" id="&lt;span class='gt_from_md'&gt;&lt;strong&gt;Characteristic&lt;/strong&gt;&lt;/span&gt;"><span class='gt_from_md'><strong>Characteristic</strong></span></th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" scope="col" id="&lt;span class='gt_from_md'&gt;&lt;strong&gt;Beta&lt;/strong&gt;&lt;/span&gt;"><span class='gt_from_md'><strong>Beta</strong></span></th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" scope="col" id="&lt;span class='gt_from_md'&gt;&lt;strong&gt;95% CI&lt;/strong&gt;&lt;/span&gt;&lt;span class=&quot;gt_footnote_marks&quot; style=&quot;white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;&quot;&gt;&lt;sup&gt;1&lt;/sup&gt;&lt;/span&gt;"><span class='gt_from_md'><strong>95% CI</strong></span><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span></th>
      <th class="gt_col_heading gt_columns_bottom_border gt_center" rowspan="1" colspan="1" scope="col" id="&lt;span class='gt_from_md'&gt;&lt;strong&gt;p-value&lt;/strong&gt;&lt;/span&gt;"><span class='gt_from_md'><strong>p-value</strong></span></th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><td headers="label" class="gt_row gt_left">age</td>
<td headers="estimate" class="gt_row gt_center"><br /></td>
<td headers="conf.low" class="gt_row gt_center"><br /></td>
<td headers="p.value" class="gt_row gt_center"><br /></td></tr>
    <tr><td headers="label" class="gt_row gt_left">    20-39</td>
<td headers="estimate" class="gt_row gt_center">—</td>
<td headers="conf.low" class="gt_row gt_center">—</td>
<td headers="p.value" class="gt_row gt_center"><br /></td></tr>
    <tr><td headers="label" class="gt_row gt_left">    40-59</td>
<td headers="estimate" class="gt_row gt_center">-4.0</td>
<td headers="conf.low" class="gt_row gt_center">-8.3, 0.40</td>
<td headers="p.value" class="gt_row gt_center">0.072</td></tr>
    <tr><td headers="label" class="gt_row gt_left">    60-99</td>
<td headers="estimate" class="gt_row gt_center">-4.7</td>
<td headers="conf.low" class="gt_row gt_center">-8.9, -0.51</td>
<td headers="p.value" class="gt_row gt_center">0.030</td></tr>
  </tbody>
  
  <tfoot class="gt_footnotes">
    <tr>
      <td class="gt_footnote" colspan="4"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height: 0;"><sup>1</sup></span> <span class='gt_from_md'>CI = Confidence Interval</span></td>
    </tr>
  </tfoot>
</table>
</div>

``` r
with(imp, lm(bmi ~ age)) %>%
  tbl_regression() %>% 
  add_glance_table()
#> Error: No glance method for objects of class mira
```

<sup>Created on 2024-08-18 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

**Comment by ddsjoberg** [comment]
Thank you for the report @CharlyMarie ! Much appreciated.

The fix for this is in PR #1914. Could you take a look ?

<img width="624" alt="image" src="https://github.com/user-attachments/assets/69b6a5ed-501d-4d2a-90e3-5e40f140ea9d">

**Comment by CharlyMarie** [comment]
Well done @ddsjoberg, it is working again :)
